### PR TITLE
add blocks to attachments

### PR DESCRIPTION
**Pull request by ykaliukh**
For my fix, I need to add the field "blocks" to the field "attachment" according to [this Slack doc](https://api.slack.com/reference/messaging/attachments#legacy_fields). For some reason, Slack API does not process requests as expected if an attachment has the field "blocks" and other fields. In order to solve this problem, a custom serializer has been added.

### CODE REVIEW

File: slack-base/src/test/resources/attachment_with_blocks.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+          "alt_text": "alt text"
+        },
+        {
+          "type": "mrkdwn",
+          "text": "<profileUrl543|name123>"
+        }
+      ]
+    }
+  ]
+}
```
Comment (zmarushchak-hs) [inline slack-base/src/test/resources/attachment_with_blocks.json:18]: New line, please.

File: slack-base/src/test/resources/attachment_without_blocks.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "footer": null,
+  "footer_icon": null,
+  "thumb_url": null,
+  "callback_id": null,
+  "actions": [],
+  "mrkdwn_in": [],
+  "ts": null
+}
```
Comment (zmarushchak-hs) [inline slack-base/src/test/resources/attachment_without_blocks.json:27]: New line, please.

File: slack-base/src/test/resources/attachment_without_blocks.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "fallback": null,
```
Comment (zmarushchak-hs) [inline slack-base/src/test/resources/attachment_without_blocks.json:2]: Do you think it makes sense to add tests with with non-null fields?

**Review by zmarushchak-hs — COMMENTED** [review]
@ykaliukh I left few comments.

**Review by zmarushchak-hs — APPROVED** [review]
@ykaliukh LGTM! But please be sure to not merge this until the end of WoR.
